portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,13 +20,15 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/*
```

Comment:
Keep this comment formatted as a Javadoc. It's required for public classes.
https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required

Reply:
Fixed!

Reply:
You need the `/**` at the start, to make this a javadoc.